src/js/components/Menu/Menu.js

```diff
@@ -57,6 +58,8 @@ To make a selection:
 - Space is pressed.
 */
 
+const defaultMenuItemsValue = [];
```

Comment:
Nit. Doesn't need to be addressed, but noticing we are deviating a bit from concise variable names where `defaultMenuItemsValue` could be `defaultItems`. Perhaps that namespace is elsewhere in the file that I'm not seeing, but we are already in the Menu file for context and value maybe extraneous.

Also caught my eye earlier in Distribution, but didn't comment there. Just noticed are style seemed to be drifting a bit.

Reply:
Good note, I'll change this to be more concise